pkg/tracing/helpers.go

```diff
@@ -17,69 +17,91 @@
 package tracing
 
 import (
-	"encoding/json"
-	"fmt"
-
 	"go.opentelemetry.io/otel/attribute"
 )
 
-func keyValue(k string, v any) attribute.KeyValue {
-	if v == nil {
-		return attribute.String(k, "<nil>")
+const (
+	// Container attributes
+	AttrContainerID      = "container.id"
+	AttrContainerName    = "container.name"
+	AttrContainerAttempt = "container.attempt"
+	AttrContainerRuntime = "container.runtime"
+
+	// Pod attributes
+	AttrPodUID              = "pod.uid"
+	AttrPodName             = "pod.name"
+	AttrPodNamespace        = "pod.namespace"
+	AttrContainerdNamespace = "containerd.namespace"
+
+	// Image attributes
+	AttrImageRef    = "container.image.ref"
+	AttrImageDigest = "image.digest"
+
+	// Infrastructure attributes
+	AttrSnapshotter = "snapshotter.name"
+	AttrNodeName    = "node.name"
+
+	// Operation attributes
+	AttrOperationType  = "operation.type"
+	AttrOperationPhase = "operation.phase"
+	AttrErrorCode      = "error.code"
+	AttrExitCode       = "container.exit_code"
+
+	// Sandbox attributes
+	AttrSandboxID   = "sandbox.id"
+	AttrSandboxName = "sandbox.name"
+)
+
+func AddStandardAttributes(span *Span, sandboxID, containerID, podName, podNS string) {
```

Comment:
It looks like it's dedicated function for CRI API. I think we can move this into `internal/cri` package.